CHANGELOG.rst

```diff
@@ -24,6 +24,8 @@ Fixed
 *****
 - ``file.get_ts_from_sentinel_filename()``: Return datetime.datetime objects instead of timestamp strings #42
 - ``raster.Image()``: in-memory dataset could not be updated if not GTiff #48
+- ``raster.Image()``: fixed init signature to consider array dimorder when creating Image from array #50
+- ``test_raster.test_arr()``: fixed incorrect validation dimorder in array #50

```

Comment:
This was never broken in a released version, I guess you don't have to write a CHANGELOG here

Reply:
Would move the changelog entries to "changed" in this case. Think we should document it somewhere and link to an issue anyway, because without this an incorrect Image instance is created.